doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -89,10 +89,10 @@ These core principles will guide NFV verification deliverables:
 <a name="1.4.1"></a>
 ### 1.4.1 Overarching Objectives and Goals
 
-1. Deliver tools and methods for verifying NFVi capabilities satisfy infrastructure needs of VNFs identified in RM.
+1. Deliver tools and methods for verifying NFVI capabilities satisfy infrastructure needs of VNFs/CNFs identified in RM.
 2. Augment current OVP program to meet infrastructure verification requirements.
```

Comment:
2 - is false. It's the opposite. RC defines requirements and test cases.
OVP is up to conform with them but RC goals are unlinked to OVP (it may be implemented by other bodies)